enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -689,9 +689,6 @@ def can_redeem(self, lms_user_id, content_key, skip_customer_user_check=False):
         # learner not associated to enterprise
         if not skip_customer_user_check:
             enterprise_user_record = self.enterprise_user_record(lms_user_id)
-            if not enterprise_user_record:
-                self._log_redeemability(False, REASON_LEARNER_NOT_IN_ENTERPRISE, lms_user_id, content_key)
-                return (False, REASON_LEARNER_NOT_IN_ENTERPRISE, [])
```

Comment:
[suggestion] Given the only thing in this code path that relies on `enterprise_user_record` is getting removed, I might recommend removing `enterprise_user_record = self.enterprise_user_record(lms_user_id)` altogether, and only rely on the `self.enterprise_user_record` call within `includes_learner`.

I believe in this current state, `self.enterprise_user_record(lms_user_id)` is potentially queried twice unnecessarily (e.g., if `self.enterprise_user_record(lms_user_id)` return `None`, `self.enterprise_user_record(lms_user_id)` is executed again, with the same result, within `self.includes_learner`).

```python
included_in_policy, reason = self.includes_learner(lms_user_id)
```

```python
def includes_learner(self, lms_user_id, enterprise_user_record=None):
  """
  Determine whether the lms user is associated properly with both the enterprise
  and the policy's group(s).
  """
  learner_record = self.enterprise_user_record(lms_user_id)
  if not learner_record:
    return False, REASON_LEARNER_NOT_IN_ENTERPRISE

  # ...
```